.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,17 +6,12 @@
 
 
 
-# FE Screenshots (If Applicable)
-
-
-
 <details>
     <summary><h4>Author Checklist</h4></summary>
 
-- [ ]  Can a reviewer skim this PR / Is this a non code change?
+- [ ]  Non code change
 - [ ]  Followed DRY (Don't repeat yourself)
-- [ ]  Migrations Required/Applied?
-- [ ]  Dependency Updates Required
-- [ ]  Tests Added
-- [ ]  Documentation Updated (If Applicable)
+- [ ]  Dependency Updates
+- [ ]  Migrations Required (& Applied)?
```

Comment:
```suggestion
- [ ]  Migrations Required
```
The relevance I see in this entry is so that when pulling the given branch locally or deploying to an env where `migrate` is not performed by default as part of the release one should remember to run `migrate`
Checking this box says 'I have added migration files'

Reply:
A fair point well made.